cdap-cli/src/main/java/co/cask/cdap/cli/util/AbstractAuthCommand.java

```diff
@@ -30,6 +35,17 @@
 
   protected final CLIConfig cliConfig;
 
+  protected static final Function<String, Set<Action>> ACTIONS_STRING_TO_SET = new Function<String, Set<Action>>() {
+    @Override
+    public Set<Action> apply(String input) {
+      ImmutableSet.Builder<Action> resultBuilder = ImmutableSet.builder();
+      for (String action : Splitter.on(",").split(input)) {
```

Comment:
Thought about that, but this is the actions string that the user enters in the CLI. Having a space in the middle will fail to parse as the CLI command. So guarding against whitespace is not necessary.